src/slice.rs

```diff
@@ -128,38 +128,52 @@ impl From<Vec<u8>> for SlicePayload {
     }
 }
 
+/// Helper function for [`create_random_slice_payload`].
+/// Creates a Vec of [`Transaction`], binencodes it, and returns the resulting byte vector.
+/// The number of transactions used and their lens are picked so that the resulting byte vector is of size `desired_size`.
+//
+// NOTE: This function is terribly hacky and making assumptions about how bincode works.
+fn create_txs(desired_size: usize) -> Vec<u8> {
```

Comment:
in general, so far I have followed the convention of ordering functions/methods from high-level (`pub`) to low-level (private), so here it would be `create_random_slice()` first, then `create_random_slice_payload()`, finally `create_txs()` at the end.